application/src/main/scala/com/azavea/franklin/error/CrudError.scala

```diff
@@ -30,41 +26,9 @@ object MidAirCollision {
   implicit val decMidAirCollision: Decoder[MidAirCollision] = deriveDecoder
 }
 
-case class InvalidPatch(msg: String, patch: Json, error: Error) extends CrudError
+case class InvalidPatch(msg: String, patch: Json) extends CrudError
```

Comment:
getting rid of the attempt to encode `Error`s solved the problem of the missing class in test runs. It also doesn't cost us much, since encoding the decoding failure / parsing failure for users isn't very nice looking. As a compromise, I threw the pretty-printed cursor ops into the error message